dashboard/db/migrate/20200221223130_create_pd_workshop_survey_foorm_submissions.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePdWorkshopSurveyFoormSubmissions < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_workshop_survey_foorm_submissions do |t|
+      t.references :foorm_submission, index: {unique: true, name: 'index_workshop_survey_submissions_on_foorm_id'}, null: false
```

Comment:
Would this now be `index_workshop_survey_foorm_submissions_on_foorm_id`?

Reply:
yep I can change it to that, I mainly just needed a shorter index name than the default

Reply:
yep I can change it to that, I mainly just needed a shorter index name than the default